src/open_inwoner/openzaak/models.py

```diff
@@ -448,6 +448,15 @@ def form_service(self, service):
         ),
         default=False,
     )
+    limit_user_visible_cases_to_role = models.CharField(
+        verbose_name=_("Betrokkene rol"),
```

Comment:
Sorry for another nitpick (I'd leave it for a separate PR, but because this has to be cherry-picked for a backport it's nice if it's in a single commit/migration): I'd like to have the verbose name reflect the function a bit better

```suggestion
        verbose_name=_("Limit cases to role"),
```